From: Paul McEnery <pmcenery@gmail.com>
Subject: Add short description to POD (slimrat and slimrat-gui)

The following patch changes the name slimrat-cli to slimrat (to match
the executable), and adds a short description of the page. It also
adds a short description to slimrat-gui, making it "apropos" and
"whatis" compliant for mandb on Unix systems.

--- a/src/slimrat
+++ b/src/slimrat
@@ -519,7 +519,7 @@
 
 =head1 NAME
 
-slimrat-cli
+slimrat - Command-line utility for downloading files
 
 =head1 VERSION
 
--- a/src/slimrat-gui
+++ b/src/slimrat-gui
@@ -629,7 +629,7 @@
 
 =head1 NAME
 
-slimrat-gui
+slimrat-gui - Graphical utility for downloading files
 
 =head1 VERSION
 
